### DISCUSSION TITLE
ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.

### DISCUSSION THREAD

**Discussion by MahapatrO** [webpack/webpack#14707] [General]
# Bug report

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
   - configuration.devtool should match pattern "^(inline-|hidden-|eval-)?(nosources-)?(cheap-(module-)?)?source-map$".
     BREAKING CHANGE since webpack 5: The devtool option is more strict.
     Please strictly follow the order of the keywords in the pattern.
   - configuration.optimization has an unknown property 'namedChunks'. These properties are valid:
     object { checkWasmTypes?, chunkIds?, concatenateModules?, emitOnErrors?, flagIncludedChunks?, innerGraph?, mangleExports?, mangleWasmImports?, mergeDuplicateChunks?, minimize?, minimizer?, moduleIds?, noEmitOnErrors?, nodeEnv?, portableRecords?, providedExports?, realContentHash?, removeAvailableModules?, removeEmptyChunks?, runtimeChunk?, sideEffects?, splitChunks?, usedExports? }
     -> Enables/Disables integrated optimizations.
     Did you mean optimization.chunkIds: "named" (BREAKING CHANGE since webpack 5)?
      at validate (/codebuild/output/src/src/backend/node_modules/webpack/node_modules/schema-utils/dist/validate.js:105:11)
      at validateSchema (/codebuild/output/src/src/backend/node_modules/webpack/lib/validateSchema.js:78:2)

**Other relevant information:**
webpack version:  5.63.0
Node.js version: 14
Operating System: W-10
Additional tools: webpack-node-externals,"@babel/preset-env": "^7.16.0",
    "@babel/preset-typescript": "^7.16.0","babel-jest": "^26.6.3",
    "babel-loader": "^8.2.3",

**Comment by alexander-akait** [comment]
Please read the error message and fix your configuration

**Reply by alexander-akait** [reply]
`namedChunks` was removed, please read https://webpack.js.org/blog/2020-10-10-webpack-5-release/

> MIGRATION: Use chunkIds and moduleIds
